tests/pages/course-page.js

```diff
@@ -55,8 +55,7 @@ export default create({
       changedFiles: collection('[data-test-community-solution-changed-file]'),
 
       unchangedFiles: collection('[data-test-community-solution-unchanged-file]', {
```

Comment:
@VasylMarchuk if you change this to `unchangedFiles: collection('[data-test-community-solution-unchanged-file]', FileContentsCard)`, does that help avoid the extra wrapper element?

Reply:
Trying it now! I don't remember the exact situation where I've had a problem, let me try again and see if I can avoid the wrapper, didn't spend much time on this last time.

Reply:
It worked well, wrapper removed!